api/v2/views/tag.py

```diff
@@ -1,30 +1,23 @@
-from rest_framework import viewsets
-from rest_framework.permissions import IsAuthenticatedOrReadOnly
-from rest_framework import serializers
 from core.models import Tag
+
+from api.permissions import CloudAdminRequired
 from api.v2.serializers.summaries import TagSummarySerializer
-from api.v2.permissions import IsAdminOrReadOnly
+from api.v2.views.base import AuthReadOnlyViewSet
 
 
-class TagViewSet(viewsets.ModelViewSet):
+class TagViewSet(AuthReadOnlyViewSet):
```

Comment:
This should be CloudAdminOrReadOnly?  Expected behavior:

Anyone can view tags.
Only authenticated users can create tags.
Only admins can modify or delete tags.